### PR TITLE
First edit of release notes  for 6.10

### PR DESCRIPTION
**Pull request by GuiMacielPereira**
### Description of work
Went over release notes and converted passive voice into active voice, added images and added links to documentation pages.

### To test:

Build documentation and read over release notes for 6.10.
Check that meaning of bullet points in the release notes makes sense. Check for grammatical and formatting errors.

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

File: docs/source/release/v6.10.0/diffraction.rst [hunk 1]
```diff
@@ -10,34 +10,53 @@ Powder Diffraction
...
 New features
 ############
-.. amalgamate:: Diffraction/Powder/New_features
+- ``create_total_scattering_pdf`` in polaris script now creates a total scattering pdf name with a default format that includes the pdf type as a suffix.
+- ``create_total_scattering_pdf`` in polaris script now accepts new parameter ``pdf_output_name`` that modifies the output name of the total scattering pdf.
 
 Bugfixes
 ############
-.. amalgamate:: Diffraction/Powder/Bugfixes
+- HRPD reduction :ref:`isis-powder-diffraction-hrpd-ref` now only subtracts empty run from the vanadium when it is subtracted from the sample run (i.e. when ``subtract_empty_instrument=True``) - note other ISIS instrument will always subtract an empty run from the vanadium.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/diffraction.rst:18]: Might make more sense when rendered.
Currently:
image

```suggestion
- :ref:`HRPD reduction script <isis-powder-diffraction-hrpd-ref>` now only subtracts empty run from the vanadium when it is subtracted from the sample run (i.e. when ``subtract_empty_instrument=True``) - note other ISIS instrument will always subtract an empty run from the vanadium.
```

File: docs/source/release/v6.10.0/direct_geometry.rst [hunk 2]
```diff
@@ -10,35 +10,11 @@ General
 
 New features
 ############
-.. amalgamate:: Direct_Geometry/General/New_features
+- In :ref:`PyChop`, the phase offset for the disk chopper controller on MERLIN can now be modified through the yaml file for MERLIN. Furthermore, the default value has been changed from `1500` to `17000`.
 
 Bugfixes
 ############
-.. amalgamate:: Direct_Geometry/General/Bugfixes
+- Fixed a bug that prevented to show detectors for Let data.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/direct_geometry.rst:17]:
```suggestion
- Fixed a bug that prevented being able to `show detectors` for Let data.
```

File: docs/source/release/v6.10.0/framework.rst [hunk 3]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
 ############
-.. amalgamate:: Framework/Algorithms/New_features
+- Algorithm :ref:`algm-SaveASCII` now supports 1D :ref:`MDHistoWorkspace`.
+- Algorithm :ref:`algm-CompressEvents` is now able to combine events together in logarithmic increasing size groups.
+- Algorithm `PolarizationCorrectionWildes` now accepts input workspace order.
+- Algorithms :ref:`algm-Abins` and :ref:`algm-Abins2D` can now import phonopy .yaml data.
+  The force constants are stored in a file named ``FORCE_CONSTANTS`` or
+  ``force_constants.hdf5`` in the same directory as the YAML file.
+  This is recommended when using large force constants arrays as the
+  YAML loader can be very slow.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:20]: The spacing of the bullet points in this section is wider once built. Not a critical issue, could be to do with the 
```
The Euphonic JSON formats are convenient to create with Python
scripts, and recommended for users who wish to somehow customise or
manipulate their data before using it with Abins(2D).
```
text block?

File: docs/source/release/v6.10.0/framework.rst [hunk 4]
```diff
@@ -10,70 +10,128 @@ Algorithms
 
 New features
 ############
-.. amalgamate:: Framework/Algorithms/New_features
+- Algorithm :ref:`algm-SaveASCII` now supports 1D :ref:`MDHistoWorkspace`.
+- Algorithm :ref:`algm-CompressEvents` is now able to combine events together in logarithmic increasing size groups.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:14]:
```suggestion
- Algorithm :ref:`algm-CompressEvents` is now able to combine events together in logarithmically increasing size groups.
```

File: docs/source/release/v6.10.0/framework.rst [hunk 5]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
+- Algorithm `PolarizationCorrectionWildes` now accepts input workspace order.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:15]:
```suggestion
- Algorithm :ref:`algm-PolarizationCorrectionWildes` now accepts input workspace order.
```

File: docs/source/release/v6.10.0/framework.rst [hunk 6]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
+    the Euphonic library.
+  - ``euphonic.ForceConstants``: force constants which may be manipulated
+    in Euphonic, and will be converted to phonon modes on a q-point
+    mesh when Abins(2D) is run.
+
+  The Euphonic JSON formats are convenient to create with Python
+  scripts, and recommended for users who wish to somehow customise or
+  manipulate their data before using it with Abins(2D).
+
+- New algorithm :ref:`LoadErrorEventsNexus <algm-LoadErrorEventsNexus>` to load events from the `bank_error_events` bank of a NeXus file.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:40]:
```suggestion
- New algorithm :ref:`LoadErrorEventsNexus <algm-LoadErrorEventsNexus>` to load events from the ``bank_error_events`` bank of a NeXus file.
```

File: docs/source/release/v6.10.0/framework.rst [hunk 7]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
+- Re-implemtation of :ref:`LoadEventNexus <algm-LoadEventNexus>` when specifying the ``CompressTolerance``. This uses significantly less memory to create fewer events overall.
+  However, the execution time of ``LoadEventNexus`` itself is generally longer; workflows that benefit from ``CompressEvents`` generally run faster.
+- Algorithm :ref:`FindSXPeaks <algm-FindSXPeaks-v1>` supports new peak finding strategy ``AllPeaksNSigma``.
+  Credits to the author of SXD2001 for the idea of using NSigma as a threshold (albeit in SXD2001 the peak finding is done in 3D).
+  Gutmann, M. J. (2005). SXD2001. ISIS Facility, Rutherford Appleton Laboratory, Oxfordshire, England.
+- Algorithm :ref:`FindSXPeaks <algm-FindSXPeaks-v1>` now includes validation rules to remove spurious peaks due to noise,
+  by allowing user to provide additional arguements as below:
+  - ``MinNBinsPerPeak``, the Minimum number of bins contributing to a peak in an individual spectrum
+  - ``MinNSpectraPerPeak``, ``MaxNSpectraPerPeak`` Minimum & Maximum number of spectra contributing to a peak after they are grouped.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:49]:
```suggestion
- Algorithm :ref:`FindSXPeaks <algm-FindSXPeaks-v1>` now includes validation rules to remove spurious peaks due to noise,
  by allowing user to provide additional arguements as below:

  - ``MinNBinsPerPeak``, the Minimum number of bins contributing to a peak in an individual spectrum
  - ``MinNSpectraPerPeak``, ``MaxNSpectraPerPeak`` Minimum & Maximum number of spectra contributing to a peak after they are grouped.
```

File: docs/source/release/v6.10.0/framework.rst [hunk 8]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
 
 Bugfixes
 ############
-.. amalgamate:: Framework/Algorithms/Bugfixes
+- Algorithm :ref:`LoadNexusProcessed <algm-LoadNexusProcessed>` is now faster to load a NeXus file.
+- Algorithm ``DSFinterp``, which was deprecated, has been removed
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:54]:
```suggestion
- Algorithm ``DSFinterp``, which was deprecated, has been removed.
```

File: docs/source/release/v6.10.0/framework.rst [hunk 9]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
+    overrided the default sampling of :ref:`algm-Abins2D` instruments if set.
+  - This did not cause results to be incorrect, but sampled
+    them on a different mesh to the expected one and could limit
+    resolution.
+  - Now the value is saved and restored after use by :ref:`algm-Abins`; it can
+    still be modified by users who wish to fiddle with the :ref:`algm-Abins2D`
+    behaviour.
+
+- Algortihm :ref:`Load <algm-Load>` now loads a single file faster.
+- Fixed doctest strings for :ref:`GenerateGroupingPowder <algm-GenerateGroupingPowder>`.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:68]: Not sure this needs to be in the release notes?

File: docs/source/release/v6.10.0/framework.rst [hunk 10]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
 
-New features
-############
-.. amalgamate:: Framework/Fit_Functions/New_features
-
 Bugfixes
 ############
-.. amalgamate:: Framework/Fit_Functions/Bugfixes
+- Search box for fitting functions in Fit interface no longer shows duplicate functions.
+- Fit Function DSFinterp1DFit, which was deprecated, has been removed.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:77]:
```suggestion
- Fit Function ``DSFinterp1DFit``, which was deprecated, has been removed.
```

File: docs/source/release/v6.10.0/framework.rst [hunk 11]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
+- Function :ref:`func-CompositeFunction` will now throw an exception if ``getNumberDomains()`` is called and there is an inconsistent number of domains in any of the member functions.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:78]:
```suggestion
- :ref:`func-CompositeFunction` will now throw an exception if ``getNumberDomains()`` is called and there is an inconsistent number of domains in any of the member functions.
```

File: docs/source/release/v6.10.0/framework.rst [hunk 12]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
 
 Data Handling
 -------------
 
 New features
 ############
-.. amalgamate:: Framework/Data_Handling/New_features
+- Algorithm :ref:`algm-LoadEventAsWorkspace2D` accepts new boolean parameter ``LoadNexusInstrumentXML``. Default is *true*.
+- File search/loading will now look in instrument data cache **on IDAaaS**.
+  The instrument data cache is the directory ``/data/instrument/`` present on IDAaaS, and contains a local copy of the data archive.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:88]: I don't think it's quite correct that it "contains a local copy of the data archive", is it just the most recent three years worth of data?

File: docs/source/release/v6.10.0/framework.rst [hunk 13]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
+  .. figure::  ../../images/6_10_release/data-cache.gif
+    :align: center
+    :width: 950px
+
 
 Bugfixes
 ############
-.. amalgamate:: Framework/Data_Handling/Bugfixes
+- Algorithm :ref:`algm-Load-v1` now guarantees that properties ``LoaderName`` and ``LoaderVersion`` are set by end of algorithm.
+- Algorithm :ref:`algm-GenerateGroupingPowder` now allows that just one of the properties ``GroupingWorkspace`` and ``GroupingFilename`` be set.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:101]:
```suggestion
- Algorithm :ref:`algm-GenerateGroupingPowder` now allows just one of the properties ``GroupingWorkspace`` and ``GroupingFilename`` be set.
```

File: docs/source/release/v6.10.0/framework.rst [hunk 14]
```diff
@@ -10,70 +10,128 @@ Algorithms
...
 ------------
 
 New features
 ############
-.. amalgamate:: Framework/Data_Objects/New_features
-
-Bugfixes
-############
-.. amalgamate:: Framework/Data_Objects/Bugfixes
+- Speedup processing of IDF XML during loading when side-by-side-view-location parameter is not used.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/framework.rst:109]:
```suggestion
- Sped up processing of IDF XML during loading when ``side-by-side-view-location`` parameter is not used.
```

File: docs/source/release/v6.10.0/indirect_geometry.rst [hunk 15]
```diff
@@ -7,23 +7,32 @@ Indirect Geometry Changes
 
 New Features
 ------------
-.. amalgamate:: Indirect/New_features
+- Algorithm :ref:`algm-ISISIndirectEnergyTransfer` now has sample logs for multiple run reductions. ``run_number`` log stores the run numbers used in the reduction, separated by commas. ``multi_run_reduction`` log informs about the reduction done on chopped runs or regular runs.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/indirect_geometry.rst:10]:
```suggestion
- Algorithm :ref:`algm-ISISIndirectEnergyTransfer` now has sample logs for multiple run reductions. The ``run_number`` log stores the run numbers used in the reduction, separated by commas. The ``multi_run_reduction`` log informs about the reduction done on chopped runs or regular runs.
```

File: docs/source/release/v6.10.0/inelastic.rst [hunk 16]
```diff
@@ -7,23 +7,56 @@ Inelastic Changes
...
+- :ref:`Elwin Tab <elwin>` no longer ocasionally ignores properties ``Background Subtraction`` and ``Normalise to Lowest Temp``.
 
 
 Algorithms
 ----------
 
 New features
 ############
-.. amalgamate:: Inelastic/Algorithms/New_features
+- Algorithm `TransformToIqt` and `CalculateIqt` has checkbox with title ``EnfoceNormalization``. This option is set to ``True`` by default (no change to the current algorithms/workflow occurs). When it is set to ``False``, the ``LHSWorkspace`` from the output from ``ExtractFFTSpectrum`` is used in both branches to perform the final workspace division and the two intermediate workspace divisions are skipped.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/inelastic.rst:56]:
```suggestion
- Algorithm :ref:`algm-TransformToIqt` and :ref:`algm-CalculateIqt` has checkbox with title ``EnfoceNormalization``. This option is set to ``True`` by default (no change to the current algorithms/workflow occurs). When it is set to ``False``, the ``LHSWorkspace`` from the output from ``ExtractFFTSpectrum`` is used in both branches to perform the final workspace division and the two intermediate workspace divisions are skipped.
```

File: docs/source/release/v6.10.0/mantidworkbench.rst [hunk 17]
```diff
@@ -5,38 +5,54 @@ Mantid Workbench Changes
...
 
 Bugfixes
 --------
-.. amalgamate:: Workbench/Bugfixes
+
+- Selecting or deselecting plot curves in plot settings no longer causes an error.
+- Overplotting workspaces onto a regular Matplotlib plot no longer causes an error.
+- Dragging a table workspace onto a plot no longer causes a crash.
+- Loading files from disk no longer causes a memory leak.
+- Previously hidden plots are no longer unhidden from some plot scripts.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/mantidworkbench.rst:16]: The bullet point spacing here has also gone larger than normal

File: docs/source/release/v6.10.0/mantidworkbench.rst [hunk 18]
```diff
@@ -5,38 +5,54 @@ Mantid Workbench Changes
...
+- Plot scripts in documentation now use ``fig.show()`` instead of ``plt.show()`` to avoid hanging Workbench.
+- Fitting property browser no longer crashesk when removing a function from a nested composite function.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/mantidworkbench.rst:18]:
```suggestion
- Fit property browser no longer crashes when removing a function from a nested composite function.
```

File: docs/source/release/v6.10.0/mantidworkbench.rst [hunk 19]
```diff
@@ -5,38 +5,54 @@ Mantid Workbench Changes
...
+- Renaming a plot in the plots widget no longer redraws the plot title regardless of the Show Title setting.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/mantidworkbench.rst:19]:
```suggestion
- Renaming a plot in the plots widget no longer redraws the plot title regardless of the ``Show Title`` workbench setting.
```

File: docs/source/release/v6.10.0/reflectometry.rst [hunk 20]
```diff
@@ -7,10 +7,17 @@ Reflectometry Changes
...
+- Algorithm :ref:`algm-SaveISISReflectometryORSO` is now able to save into either the ASCII or Nexus format. The :ref:`Save tab<refl_save>` (formerly the Save ASCII tab) has been re-named and updated to offer ORSO Nexus as an additional save format:
+
+.. figure::  ../../images/6_10_release/save_orso_nexus.png
+   :width: 300px
+
+- Algorithm :ref:`algm-SaveISISReflectometryORSO` now outputs the individual angle and transmission files, flood correction files, calibration files and the reduction script in the ORSO file metadata.
+- Algorithm :ref:`algm-SaveISISReflectometryORSO` is now able to save multiple datasets to a single file. Note that as part of this change the InputWorkspace parameter has been renamed to WorkspaceList and takes a comma separated list of workspace names. Existing scripts that call the algorithm will need to be updated to use this new parameter name. The :ref:`Save ASCII tab<refl_save_ascii>` has been updated with additional checkbox options to support this new functionality when saving in the ORSO format.
```
Comment (jhaigh0) [inline docs/source/release/v6.10.0/reflectometry.rst:17]:
```suggestion
- Algorithm :ref:`algm-SaveISISReflectometryORSO` is now able to save multiple datasets to a single file. Note that as part of this change the InputWorkspace parameter has been renamed to WorkspaceList and takes a comma separated list of workspace names. Existing scripts that call the algorithm will need to be updated to use this new parameter name. The :ref:`Save tab<refl_save>` has been updated with additional checkbox options to support this new functionality when saving in the ORSO format.
```

**Review by jhaigh0 — COMMENTED** [review]
Just minor corrections. Don't commit them all individually if you want to use the git suggestions thing. If you go to the files tab you can add them all to one batch (commit).

All in all looking very nice.

File: docs/source/release/v6.10.0/framework.rst (hunk 12 above)
Comment (GuiMacielPereira) [reply docs/source/release/v6.10.0/framework.rst:88]: Yeah, you're right, I will specify that

File: docs/source/release/v6.10.0/framework.rst (hunk 3 above)
Comment (GuiMacielPereira) [reply docs/source/release/v6.10.0/framework.rst]: Yeah, it was caused by that extra indentation

File: docs/source/release/v6.10.0/framework.rst (hunk 12 above)
Comment (jhaigh0) [reply docs/source/release/v6.10.0/framework.rst:90]: I think you've got the same bullet point spacing error here as you had in the workbench docs, so probably something to do with the figure again.

**Review by jhaigh0 — APPROVED** [review]
Looking good now.
To come will be a PR writing the index file overview of this version, and one to pick up any release notes introduced during the release period.

**Pull request merged**